org.shadowsocks.qt5client.appdata.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.shadowsocks.qt5client</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <name>ShadowSocks Qt5 Client</name>
+  <summary>A native and cross-platform shadowsocks GUI client with advanced features.</summary>
+  <description>
+    <p>
+      Shadowsocks-Qt5 is a native and cross-platform shadowsocks GUI client with advanced features.
+    </p>
+  </description>
+  <launchable type="desktop-id">org.shadowsocks.qt5client.desktop</launchable>
+  <categories>
+    <category>Network</category>
+  </categories>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://music.163.com/style/web2/img/down/b2.jpg</image>
```

Comment:
I don't think that this is a correct screenshot. :)

Reply:
It is my fault, I am looking a usable screenshot for shadowsocks-qt5.

Reply:
I see that you included a local screenshot in this PR. You can use `https://raw.githubusercontent.com/flathub/org.shadowsocks.qt5client/master/screenshot.png` as the screenshot url.